.travis.yml

```diff
@@ -1,7 +1,14 @@
 language: node_js
 node_js:
-- '0.11'
-- '0.10'
+- '4'
+env:
+  - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
```

Comment:
Ug... is this really necessary? Is there another version of node that travis supports without having to go outside the standard tree?